images/nginx-drupal/drupal.conf

```
@@ -134,18 +134,12 @@ server {
     try_files $uri @drupal;
   }
 
-  ## Support for favicon. Return an 1x1 transparent GIF if it doesn't
-  ## exist.
-  location = /favicon.ico {
-    expires 30d;
-    try_files /favicon.ico @empty;
-  }
-
-  ## Return an in memory 1x1 transparent GIF.
-  location @empty {
-    expires 30d;
-    empty_gif;
+  ## Return an in memory 1x1 transparent GIF.	
+  location @empty {	
+    expires 30d;	
+    empty_gif;	
   }
 
+  include /etc/nginx/conf.d/drupal/favicon.conf
```

```suggestion
  include /etc/nginx/conf.d/drupal/favicon.conf;
```
`nginx: [emerg] invalid number of arguments in "include" directive in /etc/nginx/conf.d/app.conf:144`